DM XMLs 1.4 tag

cecille opened the pull request:
cherry-pick of https://github.com/project-chip/connectedhomeip/pull/36153 onto 1.4 branch

cecille committed:
DM XMLs 1.4 tag

jamesharrow reviewed (changes requested):
EVSE PREF should have had P removed (see new spec PR https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10597)

jamesharrow reviewed (commented):
Removing request changes

jamesharrow commented on data_model/1.4/clusters/EnergyEVSE.xml:
```diff
@@ -240,7 +240,6 @@ Davis, CA 95616, USA
     <struct name="ChargingTargetScheduleStruct">
```
EVSE PREF should not be P

cecille replied:
this has been resolved now with the latest spec scrape.

cecille committed:
Update to head

One of the other DM PRs wasn't picked over, so this scrape
also contains those changes. This was scraped from master vs.
picking.
